doozerlib/cli/images_streams.py

```diff
@@ -97,6 +107,14 @@ def images_streams_mirror(runtime, streams, only_if_missing, live_test_mode, dry
             else:
                 exectools.cmd_assert(cmd, retries=3, realtime=True)
 
+            # mirror arm64 builder and base images for CI
+            if mirror_arm:
+                arm_cmd = f'oc image mirror --filter-by-os linux/arm64 {brew_pullspec} {upstream_dest}-arm64'
```

Comment:
I'm was thinking this will fail for images that have no ARM arch (rhel-7 golang). However as it turns out, `oc image mirror` seems to just filter out missing arches (as long as the manifest is there) regardless of specifying `--skip-missing`. So this is fine. A comment that mentions this would be a good idea.

Reply:
I didn't see skip-missing will check specific arch, if there is no such arch it will skip.
https://github.com/openshift/oc/blob/master/pkg/cli/image/mirror/mirror.go#L500
https://github.com/openshift/oc/blob/master/pkg/cli/image/mirror/mirror.go#L538

Reply:
right, just thought that would make a helpful comment in the code :)